django_select2/static/django_select2/django_select2.js

```diff
@@ -51,6 +63,7 @@
   }
 
   $(function () {
-    $('.django-select2').djangoSelect2()
+    // do not trigger select2 initialization in hidden admin inline formset rows
+    $('.django-select2').not('.empty-form .django-select2').djangoSelect2()
```

Comment:
Yes, as far as I know, there is a side effect when initialization method `djangoSelect2()` is called twice on the same element. That is why I prefer to initialize empty rows only when user adds a new row, intercepting Django's dedicated signal. For the same reason, when dismissing the change popup, the script updates all elements related to the same model as the just changed one, included those in inline rows but with the exception of empty rows. This last one, if you check the commits, is one of my last changes. I realized that when I added a new inline row after dismissing the change popup, the select2 element in the new row was improperly loaded (looked duplicated and was inactive).